apps/web/src/components/CommunityInterestDialog/CommunityInterestDialog.tsx

```diff
@@ -0,0 +1,286 @@
+            {intl.formatMessage({
+              defaultMessage: "Interest in training opportunities",
+              id: "9whw8l",
+              description:
+                "Label for users interest in training opportunities for a community",
+            })}
+          </p>
+          <BoolCheckIcon
+            value={communityInterest.trainingInterest}
+            data-h2-margin-bottom="base(x1)"
+          >
+            {communityInterest.trainingInterest
+              ? intl.formatMessage({
+                  defaultMessage: "Interested in training*",
+                  id: "edqRug",
+                  description:
+                    "Message displayed when user expresses interest in training opportunities",
+                })
+              : intl.formatMessage({
+                  defaultMessage: "Not interested in training",
+                  id: "m6NZda",
+                  description:
+                    "Message displayed when a user has expressed they are not interested in training opportunities",
+                })}
+          </BoolCheckIcon>
+          <p
+            data-h2-color="base(black.light)"
+            data-h2-font-size="base(caption)"
+            data-h2-margin-bottom="base(x1)"
+          >
+            {intl.formatMessage({
+              defaultMessage:
+                "* Note that by indicating you’re interested in jobs or training opportunities, you agree to share your profile information with Government of Canada HR and recruitment staff within this community.",
+              id: "EJPdqH",
+              description:
+                "Footnote for more information on interest in job and training opportunities",
+            })}
+          </p>
+          {communityWorkStreams.length > 0 && (
+            <>
+              <p
+                data-h2-font-weight="base(700)"
+                data-h2-margin-bottom="base(x.25)"
+              >
+                {intl.formatMessage({
+                  defaultMessage:
+                    "Preferred work streams for job opportunities",
+                  id: "Nq3XrT",
+                  description:
+                    "Label for users interest in training opportunities for a community",
+                })}
+              </p>
+              <ul
+                data-h2-list-style="base(none)"
+                data-h2-padding-left="base(0)"
+              >
+                {communityWorkStreams.map((workStream) => (
+                  <li
+                    key={workStream.id}
+                    data-h2-display="base(flex)"
+                    data-h2-align-items="base(flex-start)"
+                    data-h2-gap="base(x.25)"
+                    data-h2-margin-bottom="base(x.25)"
+                  >
+                    <BoolCheckIcon
+                      value={interestedWorkStreams.includes(workStream.id)}
+                      trueLabel={intl.formatMessage({
+                        defaultMessage: "Interested in",
```

Comment:
I think this string should be moved to a `commonMessages.interestedIn` so it can be reused.